tgui/packages/tgui/interfaces/common/ElectricalPanel.tsx

```diff
@@ -90,41 +90,44 @@ const ElectricalPanelClosed = (props: BoxProps, context) => {
 const WireControl = (props: { wire: WireSpec; index: number }, context) => {
   const { data, act } = useBackend<ElectricalData>(context);
   const target = props.index + 1;
+  let boxColor = 'green';
```

Comment:
You shouldn't embed styles in the component code

Reply:
shouldn't've been changed in the first place!